lib/tracker_api/endpoints/search.rb

```diff
@@ -0,0 +1,53 @@
+module TrackerApi
+  module Endpoints
+    class Search
+      attr_accessor :client
+
+      def initialize(client)
+        @client = client
+      end
+
+      def get(project_id, query, params={})
+        raise ArgumentError, 'Valid query required to search' unless query.is_a?(String)
+
+        params.key?(:body) ? params[:body][:query] = query : params[:body] = { query: query }
+        data = client.paginate("/projects/#{project_id}/search", params)
```

Comment:
The Tracker API docs do not explicitly state the search endpoint is paginated like they do with the stories endpoint.  My guess is that the search endpoint is not paginated so you might be able to use a single `get` here.